samples/azure-quantum/variational-quantum-eigensolver/README.md

```diff
@@ -0,0 +1,21 @@
+---
+page_type: sample
+author: guenp
+description: Variational Quantum Eigensolver
+ms.author: guenp@microsoft.com
+ms.date: 05/02/2022
+languages:
+- python
+products:
+- qiskit
```

Comment:
I don't believe this is a valid product slug for the Microsoft-wide samples gallery. In particular, the Microsoft-wide docs team has only approved `azure-quantum` and `qdk` as quantum-related slugs at this time. The full list of valid product tags is listed at https://review.docs.microsoft.com/help/contribute/metadata-taxonomies?branch=main#product (internal-only).

Reply:
This is used in the getting-started sample, so I used it here for consistency.

Reply:
Ping @xinyi-joffre @achocron, what are your thoughts on this? Should qiskit be added as a valid product slug?

Reply:
We decided to remove the slug.